output.tf

```diff
@@ -0,0 +1,9 @@
+output "public_ip" {
```

Comment:
Just a follow-up question - Why do we need TF output?

Reply:
The values we see in the output are not hard coded so we can run this template as much as we'd like. We need the output to see the values generated for our resources. We can also use the TF output to run automations.